static/src/stylesheets/pasteup/publish.js

```diff
@@ -12,10 +12,34 @@ var inquirer = require('inquirer');
 var del = require('del');
 var megalog = require('megalog');
 var ora = require('ora');
+var git = require('git-rev');
+var isGitClean = require('is-git-clean');
 
 var writeFile = Promise.promisify(fs.writeFile);
 var authTokenKey = 'npm.guardian.developers.authToken';
 
+function isMasterBranch() {
+    return new Promise(function (resolve, reject) {
+        git.branch(function (branch) {
+            if (branch === 'master') {
+                resolve();
+            } else {
+                reject(new Error('notmaster'));
+            }
+        })
+    });
+}
```

Comment:
Minor comment but, given the name, I would expect this function to return a boolean